### ISSUE TITLE
/Accessibility page updates re:new requirements

### ISSUE THREAD

**Issue by amandavisconti** [scholarslab/scholarslab.github.io#1101]
Unclear if we want these to go on /accessiblity, vs. put longer text elsewhere (post? page?) and link there from /accessibility.

I started writing about the two relevant guidelines (federal access memo, federal accessibility requirements) in response to a 9/11/2024 meeting Sarah initiated about legacy projects vs. accessibility requirements incoming from federal government. It isn't done, and I'm not completely sure about 1) current practice, 2) how all staff understand current practice, 3) any changes we need to make to legacy projects or ongoing practice (so we'll want to update it once we all understand those better).

Work is on the `` branch.

**Comment by amandavisconti** [comment]
Cut accessibility-requirements-related text and pasting here:
```
<h2>Federal access & accessibility requirements</h2>
<p>While some of this section may read as about accessibility requirement compliance, we strive to fill or exceed both the letter and the spirit of these requirements. Comparing required practice to our past/current practice will help us identify places we can do better, or where another consideration outweighed a more accessible practice (meaning that decision should be revisited and reweighed).</p>

<h3>Federally-funded research: access memo</h3>
<p>A <a href="https://www.whitehouse.gov/wp-content/uploads/2022/08/08-2022-OSTP-Public-access-Memo.pdf">White House Office of Science and Technology Policy (OSTP) memo</a> was sent to all federal agencies in 2022. Then-acting OSTP director Alondra Nelson called on all federal grant makers to establish plans that “ensure free, immediate, and equitable access to federally funded research.” The memo discusses access in a broad sense, ensuring the public is aware of and can easily access the fruits of federally funded research, a goal that includes attention to web accessibility—requiring:</p>

<ol>
<li>"publications and their supporting data resulting from federally funded research [to be made] publicly accessible without an embargo on their free and public release;</li>
<li>Establish[ment of] transparent procedures that ensure scientific and research integrity is maintained in public access policies; and,</li>
<li>Coordinat[ion] with OSTP to ensure equitable delivery of federally funded research results and data."</li>
</ol>

<p>In seeking to understand these requirements as well as support their underlying goals, we also consulted the currently in-progress <a href="https://www.neh.gov/publicaccess">NEH's Public Access Plan page</a>, which is limited to peer-reviewed journal articles and a subset of research datasets. The NEH clarifies the memo's scope, noting that for NEH-funded research "because NEH funds many areas of the humanities beyond academic research, grantees who are not actively publishing work based on their awards may find the Public Access Plan has minimal impact. Indeed, the plan will primarily affect academic institutions and scholars who receive NEH awards for research and writing, and subsequently publish in peer-reviewed journals... The NEH Public Access Plan does not apply to monographs or chapters in edited volumes".</p>

<h3>Federal ADA requirements</h3>
<p>WCAG 2.1 AA compliance is federally required by April 2026. As noted above, we already voluntarily follow these guidelines or better with our current work. Therefore, our focus will be on making sure that voluntary following converts to mandatory; that all staff understand how to follow and advise on these requirements; that we include advising and discussion of these requirements in projects, consults, and collaborations going forward; and that we address legacy projects that may not fully follow these guidelines.</p>

<p>There are exceptions to certain compliance rules, including five types of exceptions that may be relevant to our work. Each has criteria. Pre-existing social media posts and archived web content do not require remediation. Minor deviations from standards might be acceptable if they do not significantly impact user experience. (Per Communications AUL, Aug-Sept 2024 Library group zoom on accessibility)</p>

<p>Section 508: #tk</p>

<h3> Our work + the federal access & ADA requirements</h3>
<p>Much of our work already meets these required guidelines, as part of our following best practices, though this is not true of all our work (especially as we move further into the past, given best practices and the Web have evolved since our 2006 creation as a research center). This section discusses work that may need added or different approaches given the new guidelines.</p>

<p>Areas of our projects needing special accessibility attention and practice are:</p>
<ul>
<li>Datasets (both as research output focus, or that are possible to pull out from e.g. web projects)</li>
<li>Site content (text, images, media)</li>
<li>Site design & functionality</li>
</ul>

<p>Our approach also differs depending on our style of involvement:</p>
<ol>
<li><strong>Projects initiated or co-initiated by SLab staff</strong></li>
<li><strong>Collaborations led by folks other than SLab staff</strong> (e.g. Praxis Fellows annual hackathons, IATH Fellowships we collaborate on, non-SLab-faculty-led projects)</li>
<li><strong>Consultation without collaborative contribution</strong> (e.g. reference consultation and informal modes of advising, mentoring, providing feedback) => advise on best practices as well as requirements, as a built-in practice; point consultees to resources such as <a href="https://accessibility.virginia.edu/">UVA'S Accessibility website</a></li>
</ol>

<p>We divide our approach by the status of a research project:</p>
<ol>
<li><strong>Past projects that have been archived</strong>: These mostly fall into
    <ul><li>a website that's no longer online, but still has online presence such as a GitHub repo => mark the repo using GH's "archive" function so it's clear the work precedes these guidelines</li>
    <li>a project that's been through, is currently in, or is intended to go into the Special Collections DH preservation pipeline as soon as it's accepting more projects => allow Special Collections to handle if/how guidelines apply; for projects waiting to enter pipeline, we could optionally add a note to the project's homepage indicating its archival and pre-guideline status</li></ul>
</li>
<li><strong>Past projects/projects without current research work, that are still live online</strong>: We can add a note to the project's homepage indicating its past, pre-guideline status; additionally, we can contact and advise the project owner on steps to make the project meet the guidelines, and which if any of these are mandated vs. simply recommended best practices for optimal project access.</li>
<li><strong>Current in-progress projects</strong>: (plan in development; more to come soon!)</li>
<li><strong>Future projects</strong>: (plan in development; more to come soon!)</li>
</ol>

<p>While these guidelines focus on outputs and activities such as writing, data, code, and websites, our research extends to a variety of formats including our public spaces and services, community design, events, and social media. We strive to bring the values driving these guidelines to all these areas.</p>
```
